queries/c_sharp/indents.scm

```diff
@@ -0,0 +1,117 @@
+[
+ (property_declaration)
+ ; (field_declaration)
+ (variable_declaration)
+ (accessor_declaration)
+
+ ; (declaration_list)
+
+ (parameter_list)
+ (type_parameter_list)
+ (accessor_list)
+ (attribute_argument_list)
+ (base_list)
+ (argument_list)
+
+ (binary_expression)
+ (expression_statement)
+ (return_statement)
+ (local_declaration_statement)
+ (local_function_statement)
+ (yield_statement)
+ (fixed_statement)
+ (unsafe_statement)
+ (labeled_statement)
+ (for_each_statement)
+ (for_statement)
+ (if_statement)
```

Comment:
Using your example codeblock, I got this result with the if statement
```cs
if (
    with()
    && many()
    && conditions()
   )
{
  foo();
}
```
I don't think the ending `)` should be there

Reply:
hm, weird I am getting this:

Reply:
Idk why but it's working like yours on my end now :D 
Probably some failed parsing somewhere up in the file